web/modules/custom/do_content_api/tests/src/Kernel/Hook/ModerationPolicyHookTest.php

```
@@ -176,6 +177,31 @@ public function testNonApiPageUnaffected(): void {
+   */
+  public function testApiPagePublishAfterAuthoringIsHonoured(): void {
+    $api_user = $this->createUser(['use content authoring api']);
+    $this->assertNotFalse($api_user);
+    $this->setCurrentUser($api_user);
+
+    // Authoring (create) is forced to draft by the policy.
+    $node = Node::create([
+      'type' => 'civictheme_page',
+      'title' => '[TEST] Authored then published',
+      'moderation_state' => 'draft',
+    ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\node\Entity\Node' in method 'testApiPagePublishAfterAuthoringIsHonoured'. (undefined)

(StaticAccess)